utils/build/docker/python/flask-poc.base.Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN pip install pymysql cryptography pyodbc
 
```

Comment:
Can you update the version number of this base image ? look for flask-poc.base in the code